src/Nancy.Authentication.Stateless/StatelessAuthentication.cs

```diff
@@ -74,16 +74,10 @@ public static void Enable(INancyModule module, StatelessAuthenticationConfigurat
 
             return context =>
             {
-                try
-                {
-                    context.CurrentUser = configuration.GetUserIdentity(context);
-                    return context.Response;
-                }
-                catch (Exception)
-                {
-                    return context.Response;
-                }
+                context.CurrentUser = configuration.GetUserIdentity(context);
+
+                return null;

```

Comment:
Because if you don't return `null` in a before-pipeline-hook, you short-circuit the pipeline and return the response directly.